### DISCUSSION TITLE
Partition Wizard not showing up

### DISCUSSION THREAD

**Discussion by beandi** [arendst/Tasmota#19050] [Q&A] [answered]
So, as some others i stumbled into this partition size problem with my nspanel (Tasmota 12.5.0) trying to upgrade to 13.0.
The recommendation was to install the partition_wizard.tapp file to change the partition sizes, but here starts the problem:
The upload works fine, i can see the tapp-file in the file system, but after a reboot the wizrad-menu option does not appear.
During reboot the following error appears:
`00:00:00.264 Projekt tasmota - NSPanel Version 12.5.0(tasmota32)-2_0_7(2023.04.17 08:12:37)`
`00:00:00.355 TAP: Loaded Tasmota App 'Partition_Wizard.tapp'`
`00:00:00.608 BRY: failed to run compiled code 'io_error' - inconsistent number of builtin objects.`

The error only shows when the tapp file is present.
Any ideas? TIA

**Comment by Jason2866** [comment]
@s-hadinger I think the Partition Wizard is built with requires which are statisfied only in v13
To convert to safeboot the Wizard should support v12.5.0 since this is the last version where Tasmota fits in the two OTA partition scheme.

**Reply by s-hadinger** [reply]
Yes. I realized that bec file needed to be generated on 12.5 to avoid this error

**Reply by sfromis** [reply, marked as answer]
For now, you can get it working by using the next older build of Partition Wizard.
https://github.com/arendst/Tasmota/raw/eb3de9c73a23e5559123dd85df70f844aaa46f33/tasmota/berry/modules/Partition_Wizard.tapp

**Reply by schwupp** [reply]
i can confirm the issue and the workaround. i'm using a shelly plus device, converted to tasmota. For v12.4 and v12.5 only the older commit of Partition Wizard is loading, but this older commit converts layout perfectly. After conversion and upgrade to Tasmota v13, the newest commit of Partition Wizard is loading.

**Reply by s-hadinger** [reply]
This is now fixed. Please use the latest from the repo:
https://github.com/arendst/Tasmota/blob/development/tasmota/berry/modules/Partition_Wizard.tapp
